executor/explain.go

```diff
@@ -111,3 +113,11 @@ func (e *ExplainExec) generateExplainInfo(ctx context.Context) (rows [][]string,
 	}
 	return e.explain.Rows, nil
 }
+
+func (e *ExplainExec) getAnalyzeExecToExecutedNoDelay() Executor {
+	if e.analyzeExec != nil && !e.executed && e.analyzeExec.Schema().Len() == 0 {
+		e.executed = true
```

Comment:
Please add some comments for this line and this method.

Reply:
done.